spec/bundler/bundler_spec.rb

```diff
@@ -228,6 +228,37 @@
     end
   end
 
+  describe "#requires_sudo?" do
+    before do
+      allow(Bundler).to receive(:which).with("sudo").and_return("/usr/bin/sudo")
+      FileUtils.mkdir_p("tmp/vendor/bundle")
+    end
+    after do
+      FileUtils.rm_rf("tmp/vendor/bundle")
+      Bundler.remove_instance_variable(:@requires_sudo_ran)
+      Bundler.remove_instance_variable(:@requires_sudo)
+    end
+    context "writable paths" do
+      it "should return false and display nothing" do
+        allow(Bundler).to receive(:bundle_path).and_return(Pathname("tmp/vendor/bundle"))
+        expect(Bundler.ui).to_not receive(:warn)
+        expect(Bundler.requires_sudo?).to eq(false)
+      end
+    end
+    context "unwritable paths" do
+      before do
+        FileUtils.touch("tmp/vendor/bundle/unwritable.txt")
+        FileUtils.chmod(0400, "tmp/vendor/bundle/unwritable.txt")
```

Comment:
I think rubocop wants you to change this to `0o400`.

Reply:
I missed CI result.